exercises/kindergarten-garden/src/test/java/KindergartenGardenTest.java

```diff
@@ -0,0 +1,153 @@
+import org.junit.Test;
+import org.junit.Ignore;
+
+import static org.junit.Assert.assertEquals;
+
+public class KindergartenGardenTest {
+
+    @Test
+    public void singleStudent() {
+        String student = "Alice";
+        String plants = "RC\nGG";
+        Plant[] expected = {Plant.RADISHES, Plant.CLOVER, Plant.GRASS, Plant.GRASS};
+
+        assertEquals(expected,
+                new KindergartenGarden(plants).getPlantsOfStudent(student));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void singleStudent2() {
+        String student = "Alice";
+        String plants = "VC\nRC";
+        Plant[] expected = {Plant.VIOLETS, Plant.CLOVER, Plant.RADISHES, Plant.CLOVER};
+
+        assertEquals(expected,
+                new KindergartenGarden(plants).getPlantsOfStudent(student));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void twoStudents() {
+        String student = "Bob";
+        String plants = "VVCG\nVVRC";
+        Plant[] expected = {Plant.CLOVER, Plant.GRASS, Plant.RADISHES, Plant.CLOVER};
+
+        assertEquals(expected,
+                new KindergartenGarden(plants).getPlantsOfStudent(student));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void oneGardenSecondStudent() {
+        String student = "Bob";
+        String plants = "VVCCGG\nVVCCGG";
+        Plant[] expected = {Plant.CLOVER, Plant.CLOVER, Plant.CLOVER, Plant.CLOVER};
+
+        assertEquals(expected,
```

Comment:
I find it most readable with each argument on a new line;
```
asserEquals(
  expected,
  new KindergartenGarden(plants).getPlantsOfStudents(student)
);
```
But that's just my preference, so feel free to ignore me if you disagree :) The only important thing is to avoid overly long lines and to have line breaks in sensible places, which you have already done :)